spec/unit/daemon_spec.rb

```diff
@@ -27,7 +27,7 @@ def run_loop(jobs)
     end
   end
 
-  let(:server) { stub("Server", :start => nil, :wait_for_shutdown => nil) }
+  let(:server) { double("Server", :start => nil, :wait_for_shutdown => nil) }
```

Comment:
It was dead code. There was one other dead stub (transpec'd to double) as well. I just added a commit removing those stubs/doubles.